### ISSUE TITLE
Flashbangs named as HeGrenade in parse_grenades()

### ISSUE THREAD

**Issue by mkhodr** [LaihoE/demoparser#47]
Hi, 

Currently flashbangs are being shown as he grenades when using parse.parse_grenades()

```
# Using event 'weapon_fire'

 -2980.493652 | -1803.335693 | broky | weapon_flashbang
-1852.103149 | -2442.039307 | karrigan | weapon_smokegrenade
-2316.190674 | -580.786133 | karrigan | weapon_flashbang
-1783.227173 | 1180.507080 | flameZ | weapon_hegrenade
-2061.872559 | 1091.684937 | flameZ | weapon_flashbang
```

```
# Using 'parse_grenades()'

-2988.12500 | -1817.50000 | broky | HeGrenade | 723
-1853.90625 | -2406.50000 | karrigan | SmokeGrenade | 726
-2313.18750 | -557.87500 | karrigan | HeGrenade | 746
-1793.68750 | 1172.40625 | flameZ | HeGrenade | 788
-2062.56250 | 1092.75000 | flameZ | HeGrenade | 805
```

**Comment by LaihoE** [comment]
There seems to be some bugs in the demo format that they are fixing atm. I think this falls into that category. I'll revisit this soon.

**Comment by LaihoE** [comment]
Some fixes were done to parse_grenades in 0.11.0. BE AWARE, seems like they fixed this exact problem in a update some days ago, so try with 0.11.0 and with a new demo, for example from faceit.

**Comment by mkhodr** [comment]
Hello! 

Thank you for the response! I will test it with new demos from professional matches and report any new inconsistencies.

awesome job on the parser btw

**Comment by LaihoE** [comment]
Seems to have been some problems, wait for 0.11.2 as it has some fixes

**Comment by LaihoE** [comment]
after some more problems this should have been fixed in 0.11.3 with new demos. Seems like something happened to demos a few days/weeks ago.

**Comment by LaihoE** [comment]
Assume this is fixed, re-open if problems appear
